src/friendly-errors-plugin.js

```diff
@@ -98,6 +99,31 @@ class FriendlyErrorsWebpackPlugin {
   }
 }
 
+function extractErrorsFromStats(stats, type) {
+  if (isMultiStats(stats)) {
+    const errors = stats.stats
+      .reduce((errors, stats) => errors.concat(extractErrorsFromStats(stats, type)), []);
+    // Dedupe to avoid showing the same error many times when multiple
+    // compilers depend on the same module.
+    return uniqby(errors, error => error.message);
+  }
+  return stats.compilation[type];
+}
+
+function getCompileTime(stats) {
+  if (isMultiStats(stats)) {
+    // Webpack multi compilations run in parallel so using the longest duration.
+    // https://webpack.github.io/docs/configuration.html#multiple-configurations
```

Comment:
Very nice!
I didn't know about multi-compilation running in parrallel. Thanks 👍